Project/src/test/java/TestFeatures.java

```diff
@@ -200,6 +200,63 @@ public void verifyTimePicker()
         assertTrue("Arrow keys not enabled", picker.getEnableArrowKeys());
         picker.setEnableArrowKeys(false);
         assertFalse("Arrow keys not disabled", picker.getEnableArrowKeys());
+        
+        assertNotNull("Picker settings were null", picker.getSettings());       
+        
+        picker.setEnabled(false);
+        assertFalse("Picker was not disabled", picker.isEnabled());
+        assertFalse("Menu component was not disabled", picker.getComponentToggleTimeMenuButton().isEnabled());
+        assertFalse("TextField component was not disabled", picker.getComponentTimeTextField().isEnabled());
+        
+        picker.setEnabled(true);
+        assertTrue("Picker was not disabled", picker.isEnabled());
+        assertTrue("Menu component was not enabled", picker.getComponentToggleTimeMenuButton().isEnabled());
+        assertTrue("TextField component was not enabled", picker.getComponentTimeTextField().isEnabled());
+        
+        picker.setTime(LocalTime.NOON);
+        assertEquals("noon local time could not be used", LocalTime.NOON, picker.getTime());
```

Comment:
Sounds good. I like smaller tests but didn’t want to break the “paradigm”.  Would breaking out a new test class for TestTimePicker be ok? Rather than just TestFeatures?

Reply:
You can create your own test class. The current test set is just the humble beginnings and I didn't want to create lots of files with only one or two tests in them. `TestFeatures` was just my pragmatic idea to have a place where contributors can be directed to for testing their newly proposed features.